### ISSUE TITLE
Title-Rare spawn in Scarlet Graveyard

### ISSUE THREAD

**Issue by f2ame5** [Atlantiss/NetherwingBugtracker#4187]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: -Rare spawn missing

**Current behaviour**: Rare bosses dont seem to spawn even after many runs.

**Expected behaviour**: suppose to be there.please recheck

**Server Revision**:  2605

**Comment by dinsfireyo** [comment]
They are RARE

**Comment by f2ame5** [comment]
Yep i know that they are rare..we did plenty of runs and not even one spawn.. i asked a gm about it and he told me that there is a chance that this is bugged and to sumbit a ticket so they check it out just in case

**Comment by Desst** [comment]
ran sm gy 8 times and only once there was a rare boss (Fallen Champion)

**Comment by Azuluminos** [comment]
I was just about to open a new issue about this topic. 

Tonight I ran my brother in law through SM 10 times in hopes of gearing his mage a bit through rarespawns. I have played wow for as long as europe's release on live and since 3 years been part of dedicated tbc projects. I have never seen a spawn rate this low. 

Usually at least 1 of them is up. I do say ''usually''. Ocassionally, 2 or even 3 are up but this is rarely the case. But to reset 10 times consecutively in a row without ANY spawns is something that in all my years of wow I have never experienced.

Perhaps you want to look into this?
